### ISSUE TITLE
Failed to execute 'measure' on 'Performance': The mark 'beforeRender' does not exist.

### ISSUE THREAD

**Issue by tushar1996** [vercel/next.js#47560]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Production
```

### Link to the code that reproduces this issue

Cannot provide the same

### To Reproduce

Unable to reproduce the Error.This was collected in Sentry.

### Describe the Bug

Crashed in non-app: ./node_modules/next/dist/client/index.js in markRenderComplete. Getting error **Failed to execute 'measure' on 'Performance': The mark 'beforeRender' does not exist.**

### Expected Behavior

This error should not come.

### Which browser are you using? (if relevant)

Chrome Version:109.0.0

### How are you deploying your application? (if relevant)

Github

**Comment by ziyadumar** [comment]
+1

**Comment by am1rb** [comment]
I faced the same issue and found the steps to reproduce it. To see the error on the browser console:

1. Make the network slow on the inspector/network tab
2. Try to load/reload a page
3. Before the page gets fully loaded, click on the browser back/forward button

As far as I understand, it happens because the user leaves a page before it gets fully loaded.

**Comment by zodman** [comment]
Doesnt reproducible in next 12.3.4

**Comment by jessebrennan** [comment]
I am seeing this error in Github Actions running a Cypress test, but could not reproduce it by running the Cypress test locally, so it seems possible that this issue may be caused by a low-resource environment.

**Comment by foobarnes** [comment]
I'm seeing this error on a Cypress test as well. Running it on AWS CodeBuild.

**Comment by kiranvj** [comment]
Got this on production in a Amazon Lightsail instance. Other details are not know. 

> Failed to execute 'measure' on 'Performance': The mark 'beforeRender' does not exist.

```
    "next": "^13.1.1",
    "next-auth": "^4.18.8",
    "next-pwa": "^5.6.0",
    "react": "18.1.0",
```

**Comment by rwieruch** [comment]
"next": "13.4.10",

I get this sometimes in development and production. It always happens if I load a page and then quickly navigate to the next page. The URL in the browser does not show the generate URL but the abstract one whenever this error shows up. So instead of seeing 

/cars/5

I see 

/cars/[carId]

in the browser's URL.

**Comment by michaelgmiller1** [comment]
Seeing this in our prod environment via Sentry. Is there any way to mitigate this error, or at least easily silence it in our logs ?

**Comment by RomkeVdMeulen** [comment]
Getting intermittend Cypress test failures because of this on our Gitlab CI.

**Comment by foobarnes** [comment]
@RomkeVdMeulen and others, a workaround I use for Cypress to eliminate this issue was to wait for elements to be visible (especially interactive elements that manipulate Next's router with `router.push`).

**Comment by huozhi** [comment]
The fix is now landed in stable version 13.5.1, please upgrade!
